### DISCUSSION TITLE
Simbrief username not recognized

### DISCUSSION THREAD

**Discussion by davidself1001** [flybywiresim/aircraft#4969] [Help]
I can use simbrief via the mcdu with my pilot id but it (nor the efb) recognize my username.  Therefore, i cannot get data from simbrief on my erb.  anyone else have this issue?

**Comment by davidself1001** [comment]
My username is my first and last name separated by a space.  Could the space be the problem?

**Comment by davidself1001** [comment]
Figured out that my username was different than my login name.  Now I can get simbrief plan on erb but mcdu only recognizes my pilot id not username/pilot id.  I can work with that for now.

**Reply by ExampleWasTaken** [reply]
The MCDU converts your username to your user ID. That way you don't have to update your username inside the MCDU should you ever change it.
